kubeclient.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'webmock', '~> 3.0.1'
   spec.add_development_dependency 'vcr'
   spec.add_development_dependency 'rubocop', '= 0.49.1'
-  spec.add_dependency 'rest-client'
+  spec.add_dependency 'rest-client', '>= 2.0.0'
```

Comment:
As discussed I'm fine with both `~> 2.0.0` or `~> 2.0` here.

Reply:
`>= 2.0.0` was wrong as it would also accept 3.0.
Made it `~> 2.0` — will allow any 2.y.z; I don't want to pin to 2.0.z without specific reasons as rest-client is widely used gem...
Tests pass locally against:
- [x] 2.0.0
- [x] 2.0.2 (current)
- [x] 2.1.0rc1

and rest-client is generally careful about backward compat so I think we can trust their semver.

I can make travis test against rest-client master in parallel to 2.0, if anyone wants.